site/en/blog/improved-pwa-offline-detection/index.md

```diff
@@ -3,26 +3,26 @@ layout: 'layouts/blog-post.njk'
 title: Improving Progressive Web App offline support detection
 description: >
   Verification of offline support has been part of the PWA installability
-  criteria since the beginning. Starting in Chrome 89, we will start to close
-  a key loophole that allowed developers to pass the check, without an offline
-  experience.
+  criteria since the beginning.
 date: 2021-02-08
+updated: 2021-04-14
 hero: image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/4dPIp42oCe3Vq3OPMIuQ.png
 alt: Screenshot of Chrome dino game.
 authors:
   - pjmclachlan
 tags:
   - chrome-89
+  - chrome-90
   - chrome-92
   - progressive-web-apps
 ---
 
-{% Aside %}
-**TL;DR:** Offline support has been part of the PWA installability criteria
-since the beginning. We are updating the offline detection logic to ensure a
-PWA actually provides an offline experience, closing the loophole used by
-some developers to meet the installability criteria. In the future,
-sites with empty `fetch` event handlers will no longer meet the criteria.
+{% Aside 'note' %}
+**Updated April 14th, 2021:** We previously announced plans to update the
+installability criteria to ensure a PWA actually provides an offline experience.
+After listening to your feedback, and discovering a number of issues, **we have
+decided to put those plans on hold**. We strongly believe providing a valid
```

Comment:
This is not a blocking request. Typography is not a substitute for good writing. You've already emphasized this by making it a note. Why not open with "Plans for updating installability criteria are on hold."